src/Core/Framework/Test/DataAbstractionLayer/Search/Parser/SqlQueryParserTest.php

```diff
@@ -0,0 +1,72 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Test\DataAbstractionLayer\Search\Parser;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepositoryInterface;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\ContainsFilter;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+class SqlQueryParserTest extends TestCase
+{
+    use KernelTestBehaviour;
```

Comment:
```suggestion
    use IntegrationTestBehaviour;
```
Please use `IntegrationTestBehaviour` to automatically rollback the database changes and clear the cache after each test run.

Reply:
Good to know that about the tests :) I will change that

Reply:
Unfortunately I have to change a use-statement as well :| I accept that suggestion later